setup.py

```diff
@@ -0,0 +1,48 @@
+from io import open
+from re import search
+from setuptools import setup, find_packages
+
+
+PATH = 'candies/cli'
+
+
+def text_of(path):
+    with open(path, 'rt', encoding='utf8') as file:
+        return file.read()
+
+
+def version():
+    text = text_of(f'{PATH}/__init__.py')
+    match = search(r"__version__ = \'(.*?)\'", text)
+
+    return match.group(1)
+
+
+def readme():
+    return text_of('README.md')
+
+
+def requirements():
+    return text_of('requirements.txt')
+
+
+setup(
+    name='candy-cli',
+    version=version(),
+    description='Tools for writing clean CLI code.',
+    long_description=readme(),
+    author='Candy Kingdom',
+    author_email='candy.kingdom.github@gmail.com',
+    url='https://github.com/candy-kingdom/cli',
+    packages=find_packages(exclude=['tests']),
+    include_package_data=True,
+    install_requires=requirements(),
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: MIT',
+        'Natural Language :: English',
+        'Programming Language :: Python :: 3.7.4',
```

Comment:
What do you think about using `3.8.0`?

Reply:
As far as I know, we can include here any number of languages, and the more we include the more compatible the library is.

Though I'm not sure how many versions we should add, and I'm using `3.7.4` as a laptop default, but we can probably go for `3.8.0` here and then decide (or use some tool), what other versions are supported by the package.

Reply:
Done!